src/server/utils/__tests__/helpers.js

```diff
@@ -4,12 +4,12 @@ import MockExpressRequest from 'mock-express-request'
 import MockExpressResponse from 'mock-express-response'
 
 describe('storageAPI', () => {
-  test('use onlyProd should return {ok:1}', done => {
+  test('use onlyProd should return {ok:1, onlyInEnv:"test"}', done => {
```

Comment:
onlyProd returning `onlyInEnv:"test"` may be confusing
What about using something like `onlyInEnv: { current: 'test', onlyIn:['production']}` or simply  `onlyInEnv: true`?

Reply:
Changes applied, ÐApp don't require changes.